vivisect/qt/funcgraph.py

```diff
@@ -544,8 +536,8 @@ def _renderMemory(self):
             self.updateWindowTitle()
 
             self._renderDoneSignal.emit()
-        except Exception, e:
-            print e
+        except Exception as e:
+            print(e)
```

Comment:
ahhhh yeah

Reply:
This is not even my final form. Next phase in this PR is to burn all these prints and convert them into logging messages.